lib/simplecov.rb

```diff
@@ -203,7 +203,10 @@ def run_exit_tasks!
 
       # Force exit with stored status (see github issue #5)
       # unless it's nil or 0 (see github issue #281)
-      Kernel.exit exit_status if exit_status && exit_status > 0
+      if exit_status && exit_status > 0
+        $stderr.printf("SimpleCov failed build with exit %d", exit_status)
```

Comment:
Since this affects all failure modes, it should probably be tested at a more fundamental level than the `minimum_coverage.feature` cuke. Will be glad to take advice from maintainers on testing this

Reply:
I don't think we need to test all permutations, that we hit this branch once and that it prints seems a good enough test for me :+1:

Reply:
A line feed would have been prettier